src/Microsoft.Tye.Core/ProjectReader.cs

```diff
@@ -258,5 +259,14 @@ private static void EvaluateProject(OutputContext output, ProjectServiceBuilder
                 return default;
             }
         }
+
+        private static string NormalizePath(string path)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return path.Replace('/', '\\');
+            }
+            return path.Replace('\\', '/');
```

Comment:
note: this isn't super right bcause `\` is a legal filename character in POSIX. There isn't really a good option unfortunately. I guess the answer is don't do that.